modules/frontend/frontend.go

```diff
@@ -136,7 +136,7 @@ func New(cfg Config, next http.RoundTripper, o overrides.Interface, reader tempo
 	queryRangePipeline := pipeline.Build(
 		[]pipeline.AsyncMiddleware[combiner.PipelineResponse]{
 			multiTenantMiddleware(cfg, logger),
-			newAsyncQueryRangeSharder(reader, o, cfg.Metrics.Sharder, logger),
+			newAsyncQueryRangeSharder(reader, o, cfg.Metrics.Sharder, cfg.Metrics.RF1ReadPath, cfg.Metrics.Exemplars, logger),
```

Comment:
What is the rationale to move the rf1 read path setting?  It seemed ok where it was, instead of passing an extra param.

Reply:
I thought it was cleaner this way, but it's completely out of the scope of this PR, so I'll leave it out if we don't agree.